lib/FrameManager.js

```diff
@@ -87,6 +87,14 @@ class FrameManager extends EventEmitter {
     return Array.from(this._frames.values());
   }
 
+  /**
+   * @param {!string} frameId
+   * @return {!Frame}
```

Comment:
there might be no frame for the frameId

```js
/**
 * @param {!string} frameId
 * @return {?Frame}
 */
```

Reply:
nice catch!
https://github.com/GoogleChrome/puppeteer/pull/1646/commits/2ea00a341f7867a57ececf5aaec22f260fcf6217

Reply:
I'm not sure why, but this doesn't seem to be addressed yet (though I see you commit that addresses the comment)